registration-service/src/main/resources/application.yaml

```
@@ -0,0 +1,17 @@
+spring:
+  application:
+    name: registration-service
+
+server:
+  port: 8085
+
+eureka:
+  client:
+    service-url:
+      defaultZone: ${EUREKA_SERVER_URL:http://localhost:8761/eureka/}
+
+keycloak:
+  realm: ${REALM:taxi_realm}
+  server-url: ${KEYCLOAK_SERVER_URL}
+  admin-client-id: ${ADMIN_ClIENT_ID:admin-cli}
+  admin-client-secret: ${ADMIN_CLIENT_SECRET:rNATOcv9HUDGj9f7fEP58TzvZ3xIKB5H}
```

Если значение обязательно должно передаваться через переменные среды, то значение по-умолчанию указывать не следует, чтобы реализовать подход  fail fast, если значение не было передано.
Конфиденциальные данные (ключи, пароли) не стоит указывать явно.